Maple2.File.Parser/Xml/Table/WeddingPackage.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Xml.Serialization;
+using M2dXmlGenerator;
+
+namespace Maple2.File.Parser.Xml.Table;
+
+// ./data/xml/table/weddingpackage.xml
+[XmlRoot("ms2")]
+public partial class WeddingPackageRoot {
+    [M2dFeatureLocale(Selector = "id")] private IList<WeddingPackage> _package;
+}
+
+public partial class WeddingPackage : IFeatureLocale {
```

Comment:
**suggestion (bug_risk):** Consider implementing null checks for collections

In the WeddingPackage and WeddingHall classes, consider initializing the collection properties (like weddingHall and weddingItem) to empty collections in the constructor. This can prevent potential null reference exceptions when these collections are accessed.